lib/Chunk.js

```diff
@@ -13,18 +13,59 @@ const ERR_CHUNK_ENTRY = "Chunk.entry was removed. Use hasRuntime()";
 const ERR_CHUNK_INITIAL =
 	"Chunk.initial was removed. Use canBeInitial/isOnlyInitial()";
 
+/** @typedef {import("./Module.js")} Module */
+/** @typedef {import("./ChunkGroup")} ChunkGroup */
+/** @typedef {import("./ModuleReason.js")} ModuleReason */
+/** @typedef {import("webpack-sources").Source} Source */
+
+/**
+ *  @typedef {Object} Identifiable an object who contains an identifier function property
+ *  @property {() => string} identifier the resource or unique identifier of something
+ */
+
+/** @typedef {Object} WithId an object who has an id property
+ *  @property {string} id - the id of the object
```

Comment:
I think ` - ` prefix in description is extra

Reply:
Definitely